addons/rename-broadcasts/userscript.js

```diff
@@ -149,11 +181,18 @@ export default async function ({ addon, msg, console }) {
     const promptText = msg("RENAME_BROADCAST_TITLE", { name: oldName });
     const promptDefaultText = oldName;
 
-    Blockly.prompt(
+    const blocksWrapper = document.querySelector("[class*='gui_blocks-wrapper_']");
+    let reactInternalInstance = blocksWrapper[addon.tab.traps.getInternalKey(blocksWrapper)];
+    while (!reactInternalInstance.stateNode?.ScratchBlocks) {
+      reactInternalInstance = reactInternalInstance.child;
+    }
+    const blocksComponent = reactInternalInstance.stateNode;
+
+    blocksComponent.handlePromptStart(
       promptText,
       promptDefaultText,
       function (newName) {
-        newName = Blockly.Variables.trimName_(newName);
+        newName = newName.trim();
```

Comment:
`trimName_()` doesn't exist on new Blockly. It just calls [`goog.string.trim()`](https://google.github.io/closure-library/api/goog.string.html#trim), which does the same thing as the standard `trim()` method, so replacing it with `trim()` shouldn't cause issues.